src/kicker/main.c

```diff
@@ -317,21 +332,22 @@ ISR(PCINT0_vect) {
  * (When CS00 is one, and CS01 is 0, no prescale. We don't use this)
  */
 ISR(TIMER0_COMPA_vect) {
+    sei();
```

Comment:
Uhhh. This enables interrupts. I think

Reply:
Yup, that's how you enable nested interrupts, we were trying to allow spi communication to interrupt the timer to prevent a separate issue